contrib/drivers/ceph/ceph_metrics.go

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2019 The OpenSDS Authors.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License"); you may
+//    not use this file except in compliance with the License. You may obtain
+//    a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//    License for the specific language governing permissions and limitations
+//    under the License.
+package ceph
+
+import (
+	log "github.com/golang/glog"
+	"github.com/opensds/opensds/pkg/model"
+	"gopkg.in/yaml.v2"
+	"strconv"
+	"time"
+)
+
+// TODO: Move this Yaml config to a file
+var data = `
+resources:
+  - resource: pool
+    metrics:
+      - pool_used_bytes
+      - pool_raw_used_bytes
+      - pool_available_bytes
+      - pool_objects_total
+      - pool_dirty_objects_total
+      - pool_read_total
+      - pool_read_bytes_total
+      - pool_write_total
+      - pool_write_bytes_total
```

Comment:
Can we keep consistent metrics naming style?
https://github.com/opensds/opensds/blob/master/contrib/drivers/lvm/lvm_metrics.go#L46

Is there any reason this has to adopt a different style?

Reply:
Ok, this is fine.